src/w3c/templates/headers.js

```diff
@@ -5,13 +5,25 @@ import showLink from "./show-link";
 
 export default conf => {
   const html = hyperHTML;
+  const specTitleElem = document.querySelector("h1#title, body>h1") || document.createElement("h1") ;
+  if (specTitleElem.parentElement) { // remove it from the DOM tree, because we are going to relocate it
+    specTitleElem.remove();
+    specTitleElem.innerText.trim();
+  } else {
+    specTitleElem.innerText = conf.title;
+  }
+  decorateSpecTitle(specTitleElem);
+  function decorateSpecTitle(specTitleElem) {
+    specTitleElem.classList.add("title", "p-name");
+    specTitleElem.id = "title";
+  }
   return html`<div class='head'>
   ${conf.logos.map(showLogo)}
-  <h1 class='title p-name' id='title'>${conf.title}</h1>
+  ${specTitleElem}
   ${conf.subtitle ? html`
-    <h2 id='subtitle'>${conf.subtitle}</h2>
+    <h2 property='${conf.doRDFa ? "bibo:subtitle" : null}' id='subtitle'>${conf.subtitle}</h2>
```

Comment:
nit: seems your rebase has re-introduced the `doRDFa` stuff... we recently removed that.